source/integration/references/compile.rst

```diff
@@ -76,17 +76,30 @@ where *<install\_options>* can be one or more of:
 | **-h** | prints installer help                                                                                                                                                        |
 +--------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+.. note:: 
+    
+    If you choose the ``system-wide`` installation, OpenNebula will be installed in the following folders: 
+        -   /etc/one 
+        -   /share/docs/one
```

Comment:
I believe this path is incorrect

Reply:
Yes, should be /usr/share/docs/one I think.